alot/commands/__init__.py

```diff
@@ -22,6 +22,20 @@ def __init__(self):
         self.undoable = False
         self.help = self.__doc__
 
+        mode, cmdname = reverse_lookup_command(self.__class__)
```

Comment:
> it would probably be best to just store the result for both class and the name

Do you mean that it would be better to leave the `mode` out? That would make the lookup ambiguous since the same command name can exist in different modes. Or did I misunderstand?